samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SavePublishedSettingsListener.java

```diff
@@ -497,7 +497,7 @@ else if(!assessmentSettings.getIsValidFeedbackDate()){
 				error=true;
 			}
 
-			if(StringUtils.isNotBlank(assessmentSettings.getFeedbackEndDateString()) && assessmentSettings.getFeedbackDate().after(assessmentSettings.getFeedbackEndDate())){
+			if(assessmentSettings.getFeedbackDate() == null && StringUtils.isNotBlank(assessmentSettings.getFeedbackEndDateString()) && assessmentSettings.getFeedbackDate().after(assessmentSettings.getFeedbackEndDate())){
```

Comment:
This doesn't make sense to me. Why check if the date is null and then use that date in the after call?

Reply:
The error on Jira is caused by a null pointer as feedbackDate, so i think that we need to ensure that it is not null before access it.